src/daycare.c

```diff
@@ -898,7 +902,7 @@ static void _GiveEggFromDaycare(struct DayCare *daycare)
     AlterEggSpeciesWithIncenseItem(&species, daycare);
 #endif
     SetInitialEggData(&egg, species, daycare);
-    InheritIVs(&egg, daycare);
+    InheritIVs(&egg, daycare, &daycare->mons[parentSlots[1]].mon, &daycare->mons[parentSlots[0]].mon);
```

Comment:
`daycare` is being passed already, so there's no need to pass the individual parents, like in `AlterEggSpeciesWithIncenseItem`.